pkg/config/config.go

```diff
@@ -1219,6 +1257,18 @@ func InitializeConfig(confPath string, configCheck, configStrict bool, enforceCm
 			os.Exit(1)
 		}
 	}
+
+	// Load Security Enhanced Mode configuration file
+	semConfigPath := cfg.Security.SEMPath
+	if semConfigPath != "" {
+		config, err := loadSEMConfig(semConfigPath)
+		if err != nil {
+			err = fmt.Errorf("sem configuration loading failed: %w", err)
+		}
+		terror.MustNil(err)
```

Comment:
> Do we have some way to verify whether the SEM configuration file is correct?
> 
> If the user write some wrong field of the SEM and cause their cluster unable to start up, they'll feel unhappy.

We have detection function that checks if the system variables filled in the SEM configuration file exist, it's in pkg/sessionctx/variable/variable.go，func IsVarExists(name string)

Reply:
After the SEM configuration is deserialized in the pkg/config/config.go file, it will immediately undergo validity validation at the isValidSEMConfig method.

Reply:
> After the SEM configuration is deserialized in the pkg/config/config.go file, it will immediately undergo validity validation at the isValidSEMConfig method.

After the SEM configuration is deserialized in the pkg/config/config.go file, it will immediately undergo validity validation at the isValidSEMConfig method.